perception_manager/TabletopObject[] array
int8 yumi_status # Reports the latest status of yumi. 0=latest task was successful and yumi is available, 1=yumi is busy, -2=latest task was failure and yumi is available
